fix(cli): npm package v9.79.0 missing chunk-*.js files, CLI completely broken

## Bug Description

CLI v9.79.0 is completely broken. Both `vm0` and `zero` binaries fail with `ERR_MODULE_NOT_FOUND` because the npm package is missing the `chunk-*.js` file that `index.js` and `zero.js` depend on.

**This is P0** — users who upgraded to v9.79.0 cannot use the CLI at all, and `vm0 upgrade` itself is broken so they cannot self-recover.

## Error

```
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '.../chunk-WGCV3IBM.js'
imported from .../index.js
```

## Root Cause

In `turbo/apps/cli/package.json`, the `postbuild` script sets `this.files=["."]` in the dist `package.json`. npm's `files` field does not correctly include chunk files when using `"."` as a pattern — it only packs the bin entry files (`index.js`, `zero.js`) and `package.json`.

**v9.79.0 npm tarball contents** (missing chunk):
```
package/index.js
package/package.json
package/zero.js
```

**Expected** (should include chunk):
```
package/chunk-WGCV3IBM.js
package/index.js
package/package.json
package/zero.js
```

## Fix

Change `this.files=["."]` to `this.files=["*.js","*.js.map"]` in the `postbuild` script of `turbo/apps/cli/package.json`.

## Verification

After fix, run `pnpm build` in `turbo/apps/cli/` and verify `chunk-*.js` is listed in `npm pack --dry-run` output from the `dist/` directory.

## Impact

- All users on v9.79.0 have a completely broken CLI
- `vm0 upgrade` is also broken, so users must manually run `npm install -g @vm0/cli@9.78.0` to downgrade
- Needs immediate patch release after fix

--- a/.github/workflows/turbo.yml
+++ b/.github/workflows/turbo.yml
@@ -1091,6 +1091,29 @@ jobs:
         run: cd turbo && pnpm install --frozen-lockfile --filter @vm0/cli
       - name: Build CLI
         run: cd turbo && pnpm --filter @vm0/cli build
+      - name: Verify CLI package contents
+        run: |
+          cd turbo/apps/cli/dist
+          # Ensure all imported chunks are present in the package
+          for f in index.js zero.js; do
+            grep -oP './chunk-[A-Z0-9]+\.js' "$f" 2>/dev/null || continue
+            grep -oP './chunk-[A-Z0-9]+\.js' "$f" | while read chunk; do
+              chunk_file="${chunk#./}"
+              if [ ! -f "$chunk_file" ]; then
+                echo "ERROR: $f imports $chunk but file is missing from dist/"
+                exit 1
+              fi
+            done
+          done
+          # Verify npm pack includes all JS files
+          npm pack --dry-run 2>&1 | tee /tmp/pack-list.txt
+          for js in *.js; do
+            if ! grep -q "$js" /tmp/pack-list.txt; then
+              echo "ERROR: $js exists in dist/ but would not be included in npm package"
+              exit 1
+            fi
+          done
+          echo "CLI package verification passed"
       - name: Deploy CLI with production dependencies
         run: cd turbo && pnpm deploy --filter @vm0/cli --prod --legacy --config.node-linker=hoisted /tmp/cli-deploy
       - name: Upload CLI artifact

--- a/.github/workflows/turbo.yml
+++ b/.github/workflows/turbo.yml
@@ -1093,11 +1093,12 @@ jobs:
         run: cd turbo && pnpm --filter @vm0/cli build
       - name: Verify CLI package contents
         run: |
+          set -euo pipefail
           cd turbo/apps/cli/dist
           # Ensure all imported chunks are present in the package
           for f in index.js zero.js; do
-            grep -oP './chunk-[A-Z0-9]+\.js' "$f" 2>/dev/null || continue
-            grep -oP './chunk-[A-Z0-9]+\.js' "$f" | while read chunk; do
+            chunks=$(grep -oP './chunk-[A-Z0-9]+\.js' "$f" 2>/dev/null || true)
+            for chunk in $chunks; do
               chunk_file="${chunk#./}"
               if [ ! -f "$chunk_file" ]; then
                 echo "ERROR: $f imports $chunk but file is missing from dist/"

--- a/.github/workflows/turbo.yml
+++ b/.github/workflows/turbo.yml
@@ -1092,6 +1092,7 @@ jobs:
       - name: Build CLI
         run: cd turbo && pnpm --filter @vm0/cli build
       - name: Verify CLI package contents
+        shell: bash
         run: |
           set -euo pipefail
           cd turbo/apps/cli/dist

--- a/turbo/apps/cli/package.json
+++ b/turbo/apps/cli/package.json
@@ -18,7 +18,7 @@
   ],
   "scripts": {
     "build": "tsup",
-    "postbuild": "cp package.json dist/ && pnpm json -I -f dist/package.json -e 'this.name=\"@vm0/cli\"; delete this.private; delete this.scripts; delete this.devDependencies; delete this.dependencies[\"@vm0/core\"]; this.bin[\"vm0\"]=\"index.js\"; this.bin[\"zero\"]=\"zero.js\"; this.files=[\".\"]'",
+    "postbuild": "cp package.json dist/ && pnpm json -I -f dist/package.json -e 'this.name=\"@vm0/cli\"; delete this.private; delete this.scripts; delete this.devDependencies; delete this.dependencies[\"@vm0/core\"]; this.bin[\"vm0\"]=\"index.js\"; this.bin[\"zero\"]=\"zero.js\"; this.files=[\"*.js\",\"*.js.map\"]'",
     "dev": "tsup --watch",
     "test": "vitest run",
     "test:watch": "vitest",

